packaging/suse/portusctl/lib/configurator.rb

```diff
@@ -50,9 +50,10 @@ def ssl
         "-e", @options["ssl-email"],
         "-c", @options["ssl-country"],
         "-l", @options["ssl-city"],
-        "-s", @options["ssl-state"]
+        "-s", @options["ssl-state"],
+        "-x", @options["digest"]
       ]
-      Runner.exec("gensslcert", args)
+      Runner.exec("/srv/Portus/packaging/suse/portusctl/bin/gensslcert", args)
```

Comment:
@Salzig It's `portusctl`, which is the utility installed with the RPM. In this scenario it's guaranteed that Portus is installed in  `/srv/Portus`.

@jloehel  In order to be cleaner, you should be using the `PORTUS_ROOT` constant as defined [here](https://github.com/SUSE/Portus/blob/master/packaging/suse/portusctl/lib/constants.rb).